env.js

```diff
@@ -43,7 +44,7 @@ const defaults = {
   OC_ENV: process.env.NODE_ENV || 'development',
   OC_SECRET: crypto.randomBytes(16).toString('hex'),
   WISE_ENVIRONMENT: 'sandbox',
-  API_PROXY: true,
+  API_PROXY: false,
```

Comment:
As I understand. this mean off in all environments unless API_PROXY is set to true. Practically API proxy off for dev and ci environment, it was turned on before.

Reply:
If that works, follow up is to fully remove the API proxy?